content/applications/websites/livechat/overview/responses.rst

```diff
@@ -1,48 +1,137 @@
+         :align: center
+         :alt: View of the message generated from using the /help command in Odoo Live Chat.
+
+   .. tab:: `/helpdesk`
+
+      The `/helpdesk` command uses the conversation to create a :guilabel:`Helpdesk Ticket`. The
+      transcript from the conversation will be added to the new ticket, under the
+      :guilabel:`Description` tab.
+
+      .. image:: responses/live-chat-helpdesk.png
+         :align: center
+         :alt: View of the results from a helpdesk search in a Live Chat conversation.
+
+      See :doc:`Start receiving tickets
+      </applications/services/helpdesk/overview/receiving_tickets>` for more information.
+
+   .. tab:: `/helpdesk_search`
+
+      The `/helpdesk_search` command allows an operator to search through :guilabel:`Helpdesk`
+      tickets by ticket number or keyword.
+      If a related ticket is found, a link will be generated in the conversation window.
+
+      .. image:: responses/live-chat-helpdesk-search.png
+         :align: center
+         :alt: View of the results from a helpdesk search in a Live Chat conversation.
+
+   .. tab:: `/history`
+
+      The `/history` command generates a list of the most recent pages the visitor has viewed on
+      the website (up to 15).
+
+      .. image:: responses/live-chat-responses-history.png
+         :align: center
+         :alt: View of the results from a /history command in a Live Chat conversation.
+
+   .. tab:: `/lead`
+
+      This command creates a :guilabel:`Lead` in the :guilabel:`CRM` application. After typing
+      `/lead`, create a title for this new lead, then press `Enter`. The transcript of the
+      conversation is added to the :guilabel:`Internal Notes` tab of the lead form. On the
+      :guilabel:`Extra Information` tab of the lead form, the :guilabel:`Source` will be listed as
+      :guilabel:`Livechat`.
+
+      .. image:: responses/live-chat-responses-lead.png
+         :align: center
+         :alt: View of the results from a /lead command in a Live Chat conversation.
+
+   .. tab:: `/leave`
+
+      The `/leave` command lets the *operator* leave the conversation channel.
 
 .. important::
-   | - For *helpdesk tickets*: make sure the application is installed on your database and the
-     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`
-     is enabled.
-   | - For *leads*: the *CRM* application must be installed on your database.
+   - The `/helpdesk` and `/helpdesk_search` commands can only be used if the :guilabel:`Helpdesk`
+     app has been installed, and :guilabel:`Live Chat` has been activated on a :guilabel:`Helpdesk
+     Team`. To activate :guilabel:`Live Chat`, go to :menuselection:`Helpdesk --> Configuration -->
+     Teams`, and select a team. Scroll to the :guilabel:`Channels` section and check the box
+     labeled :guilabel:`Live Chat`.
+   - The `/lead` command can only be used if the :guilabel:`CRM` app has been installed.
```

Comment:
This might be more useful beneath the breakdown of this specific command, as a note